app/src/main/res/values/strings.xml

```diff
@@ -778,4 +778,31 @@
     <string name="main_drawer_close">Close</string>
     <string name="main_drawer_help">Help</string>
     <string name="main_drawer_login">Log in / join Wikipedia</string>
+    <string name="split_large_lists_dialog_ok_button_text">OK</string>
+    <string name="protected_page_warning_dialog_ok_button_text">OK</string>
+    <string name="account_editing_blocked_dialog_ok_button_text">OK</string>
+    <string name="custom_date_picker_dialog_ok_button_text">OK</string>
+    <string name="edit_conflict_dialog_ok_button_text">OK</string>
+    <string name="feed_lang_selection_dialog_ok_button_text">OK</string>
+    <string name="customize_lang_selection_dialog_ok_button_text">OK</string>
+    <string name="reverted_edit_dialog_ok_button_text">OK</string>
+    <string name="remove_from_list_dialog_ok_button_text">OK</string>
+    <string name="remove_language_dialog_ok_button_text">OK</string>
+    <string name="remove_all_language_warning_dialog_ok_button_text">OK</string>
+    <string name="text_input_dialog_ok_button_text">OK</string>
+    <string name="delete_list_dialog_ok_button_text">OK</string>
+    <string name="ok">OK</string>
+    <string name="text_input_dialog_cancel_button_text">Cancel</string>
+    <string name="account_editing_blocked_dialog_cancel_button_text">Cancel</string>
+    <string name="feed_lang_selection_dialog_cancel_button_text">Cancel</string>
+    <string name="customize_lang_selection_dialog_cancel_button_text">Cancel</string>
+    <string name="log_out_dialog_cancel_button_text">Cancel</string>
+    <string name="editing_login_cancel_button_text">Cancel</string>
+    <string name="remove_from_list_dialog_cancel_button_text">Cancel</string>
+    <string name="remove_from_offline_cancel_button_text">Cancel</string>
+    <string name="download_using_mobile_data_cancel_button_text">Cancel</string>
+    <string name="remove_language_dialog_cancel_button_text">Cancel</string>
+    <string name="custom_date_picker_dialog_cancel_button_text">Cancel</string>
+    <string name="delete_list_dialog_cancel_button_text">Cancel</string>
+    <string name="cancel">Cancel</string>
```

Comment:
Is it possible that have these "OK" items having the same naming logic for the related components?

For example, the `delete_list_dialog_ok_button_text` should be named as `reading_list_delete_confirm_ok_button_text`, and put it under the `reading_list_delete_confirm` item.

```
    <string name="reading_list_added_articles_list_titled">Added %1$d articles to %2$s</string>
    <string name="reading_list_delete_confirm">Are you sure you want to delete %s?</string>
    <string name="reading_list_delete_confirm_ok_button_text">OK</string>    <=== Like this
```

Thank you.

Reply:
Done.